howto/android/set-automotive-properties.md

```diff
@@ -17,21 +17,85 @@ This how-to guide explains the steps of how to set the automotive properties whe
 
 ## Prerequisites
 
-- Install and initialize Anbox Cloud. Register with the dashboard and log in with your Ubuntu SSO account. {ref}`tut-installing-appliance` shows how to get this step done.
-- Create an automotive application and stream it. {ref}`tut-create-virtual-device` shows how to create an application. Remember to choose *Automotive* as your application type.
+Install and initialize Anbox Cloud. Register with the dashboard and log in with your Ubuntu SSO account. Follow {ref}`tut-installing-appliance` to do this in the appliance.
 
-## Set values using the VHAL panel
+You need an automotive application that you can stream. {ref}`tut-create-virtual-device` shows how to create an application. Remember to choose *Automotive* as your application type.
 
-Now, you are streaming the AAOS application. The controls on the right help you set specific automotive properties.
+When you have an AAOS application streaming, you can adjust the automotive properties using the dashboard or the API.
 
-Now, try setting the temperature of the automotive:
+::::{tab-set}
+:::{tab-item} API
+:sync: api
 
-1. Open the HVAC panel (fan icon in the navigation bar at the bottom).
-1. Select *All VHAL properties* on the right.
-1. Search for *HVAC_TEMPERATURE_DISPLAY_UNITS* which indicates the unit of temperature.
-1. By default, the HVAC panel displays the temperature in Fahrenheit while the VHAL properties show values in Celsius. To switch the HVAC panel display to Celsius, update the value from 49 to 50.
-1. Now, search for *HVAC_TEMPERATURE_SET*. Modify the temperatures for *Area 1* and *Area 2* and *Save*.
+You can adjust the automotive properties with the {ref}`Anbox HTTP API <sec-anbox-https-api-vhal>`.
 
-You can learn more about the various [supported system properties in the VHAL](https://source.android.com/docs/automotive/vhal/system-properties) and try changing them.
+Inside the instance terminal, install `curl` to send HTTP requests using the API:
+
+  sudo apt install curl
+
+Install `jq` so that you can parse JSON responses better:
+
+  sudo apt install jq
+
+Check the value set for the temperature property currently and observe the response:
+
+  curl -s -X GET --unix-socket /run/user/1000/anbox/sockets/api.unix s/1.0/vhal/0x15600503/0x31 | jq .
+
+The output could look like:
+
+```
+{
+  "metadata": {
+    "value": {
+      "area_id": 49,
+      "float_values": [
+        16
+      ],
+      "prop": 358614275,
+      "status": 0,
+      "timestamp": 0
+    }
+  },
+  "status": "Success",
+  "status_code": 200,
+  "type": "sync"
+}
+```
```

Comment:
Should we mention (in passing) why we have a difference between the prop IDs?
We could also use decimal values everywhere instead of hexadecimal:
````suggestion
Check the value set for the temperature property currently and observe the response:

  curl -s -X GET --unix-socket /run/user/1000/anbox/sockets/api.unix s/1.0/vhal/358614275/49 | jq .

The output could look like:

```
{
  "metadata": {
    "value": {
      "area_id": 49,
      "float_values": [
        16
      ],
      "prop": 358614275,
      "status": 0,
      "timestamp": 0
    }
  },
  "status": "Success",
  "status_code": 200,
  "type": "sync"
}
```
````

While reading through this, I wondered if that could lead to people asking questions as to why it's different. What do you think?